Fix field name inconsistency in Fortigate parsers for ASim tables

v-atulyadav opened the pull request:
Required items, please complete

Change(s):

  Files from PR #12794

v-atulyadav committed:
Fix field name inconsistency in Fortigate parsers for ASim tables

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-atulyadav committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-atulyadav committed:
Update ASimNetworkSessionFortinetFortiGate.yaml

v-sabiraj committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-sabiraj committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-sabiraj committed:
Add EventProduct field to Fortinet FortiGate parsers

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update Fortinet FortiGate parser versions and add sample data

v-sabiraj committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-sabiraj committed:
Update LastUpdated dates for Fortinet FortiGate parsers

v-sabiraj committed:
fixing DvcIdType colunn

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
validations

v-sabiraj committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
change datatype for sessionid

Commit:
Merge remote-tracking branch 'origin/master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update LastUpdated and add SessionId to parser

Commit:
Merge remote-tracking branch 'origin/master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update Fortinet_Fortigate_NetworkSession_IngestedLogs.csv

v-sabiraj committed:
Update sessionid type and parser metadata

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update Fortinet_Fortigate_NetworkSession_IngestedLogs.csv

v-sabiraj committed:
validations

v-sabiraj committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-sabiraj committed:
validations

v-sabiraj committed:
Update Fortinet severity mappings and sample data

Commit:
Merge remote-tracking branch 'origin/master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Refactor Fortinet FortiGate ASIMWebSession parser

v-sabiraj committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update Fortinet_FortiGate_Dns_IngestedLogs.csv

v-sabiraj committed:
validations

v-sabiraj committed:
Update Fortinet DNS parser samples and metadata

v-sabiraj committed:
validations

v-kasghosh committed:
Validations

v-kasghosh committed:
Revert "Validations"

This reverts commit 729f6f1723d0f921c1eae559523c733853373f6c.

v-kasghosh committed:
validations

Commit:
Merge remote-tracking branch 'origin/master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Revert "validations"

This reverts commit cfe6ede7c888478c39db0eaaff6f50e405aa93ab.

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
validations

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

v-kasghosh committed:
Revert "validations"

This reverts commit d32ddcec61ce0c835c03c13c17597726b5dba7fb.

v-kasghosh committed:
validations

v-kasghosh committed:
validations

v-kasghosh committed:
validations

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
validations

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
validations

v-kasghosh committed:
fixing validation error

trying to fix the validations for AssertionError: 0 != 1 : Parameter: responsecodename - Returned results for non existing filter value. Filtered by value: '!not_REAL_vAlUe'

Commit:
Merge remote-tracking branch 'origin/master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

v-kasghosh committed:
Revert "fixing validation error"

This reverts commit 8291aa53996fc2d781b789e66f58a7aaa976638c.

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

v-kasghosh committed:
validations

v-kasghosh committed:
Revert "validations"

This reverts commit d0562c8fdb72acda6257f01fc287b3b40c434da9.

v-kasghosh committed:
validations

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Revert "validations"

This reverts commit f7ad1ef037c2d0900a428b4b2024e7e65a135d35.

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
validations

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Revert "validations"

This reverts commit 79294f08013b8f511892b00f52a912fbcee0be43.

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Updated vim file

v-utpalkumar committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
update the vim file

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
File updated

v-utpalkumar committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

v-kasghosh committed:
Revert "update the vim file"

This reverts commit 6736058d6bfecd11c397c0dab09405378fae540f.

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
vimDNS file updated

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Revert "vimDNS file updated"

This reverts commit 3278cc2b593e9469140635068f32b9c2dcecf0a9.

v-utpalkumar committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-utpalkumar committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Update vimdnsfortinate

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-utpalkumar committed:
File updated to run checks

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Revert "Update vimdnsfortinate"

This reverts commit 9ed9cdde17f3568fc1e6bcfee98de67b568dbff3.

v-utpalkumar committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
vimDnsFortinetFortigate file updated

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update parser last updated dates and query logic

v-sabiraj committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Update the vimdnsfortinetfortigate yaml file

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Revert "Update the vimdnsfortinetfortigate yaml file"

This reverts commit 871c2330c7deaf3e0526490b5e9a54ba45bad50b.

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-sabiraj committed:
Update LastUpdated date and clean up parser lookup

v-sabiraj committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

vakohl commented on Parsers/ASimWebSession/Parsers/ASimWebSessionFortinetFortiGate.yaml:
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Web Session ASIM parser for Fortinet FortiGate
-  Version: '0.3.0'
```
I don't see any changes

vakohl commented on Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionFortinetFortiGate.yaml:
```diff
@@ -39,8 +39,8 @@ ParserQuery: |
-    "6", "Critical", // High
-    "7", "Alert", // Medium
+    "6", "High", // Critical
+    "7", "Medium", // Alert
```
6 as high and 7 as medium?

v-kasghosh committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-kasghosh committed:
Update Fortinet FortiGate ASIM parsers field names and severity mapping

Standardized user field names from _User/_UnauthUser to User/UnauthUser in DNS parsers. Adjusted severity mapping for Critical and Alert levels in NetworkSession parsers. Reverted version and LastUpdated fields in WebSession parsers to previous values.

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
validations

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Update severity mapping for FortiGate critical events

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Rename User fields to _User and _UnauthUser in Fortinet parsers

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Rename user fields in Fortinet DNS parsers

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
validation fixed

v-kasghosh committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

Commit:
Merge remote-tracking branch 'origin/master' into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Rolled back a modification

v-utpalkumar committed:
Merge branch 'v-atulyadav/fortigateASIM' of https://github.com/Azure/Azure-Sentinel into v-atulyadav/fortigateASIM

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-kasghosh committed:
Update severity mapping for FortiGate error level

Commit:
[ASIM Parsers] Generate deployable ARM templates from KQL function YAML files.

v-utpalkumar committed:
Merge branch 'master' into v-atulyadav/fortigateASIM

v-utpalkumar committed:
Modified the last-updated date accordingly
